armi/settings/fwSettings/globalSettings.py

```diff
@@ -406,18 +408,37 @@ def defineSettings() -> List[setting.Setting]:
         ),
         setting.Setting(
             CONF_BU_GROUPS,
-            default=[10, 20, 30, 100],
-            label="Burnup Groups",
+            default=[10, 20, 30],
+            label="Burnup XS Groups",
             description="The range of burnups where cross-sections will be the same "
             "for a given assembly type (units of %FIMA)",
             schema=vol.Schema(
                 [
                     vol.All(
-                        vol.Coerce(int), vol.Range(min=0, min_included=False, max=100)
+                        vol.Coerce(int),
+                        vol.Range(
+                            min=0,
+                            min_included=False,
+                        ),
                     )
                 ]
             ),
         ),
+        setting.Setting(
+            CONF_TEMP_GROUPS,
+            default=[],
+            label="Temperature XS Groups",
+            description="The fuel temperature boundaries where in between where cross-sections will be grouped together.",
+            schema=vol.Schema(
+                [vol.All(vol.Coerce(int), vol.Range(min=0, min_included=False))]
+            ),
+        ),
+        setting.Setting(
+            CONF_XS_TEMP_ISOTOPE,
+            default="U235",  # should work for non-thorium reactor. For Thorium change to U233 or TH232
```

Comment:
`U238` is probably the more dominant contributor to temperature dependence of cross sections in most reactors, but the temperature of U235 and U238 should always be identical so I don't think it matters.

Reply:
Would it be better to make this setting an `XSModelingOption` that could be set differently for each XS group? Then you could theoretically also have temperature groups for control/absorber materials.

Reply:
235 is where the heat is typically produced so it would swing more in that that component if there were multiple enrichments.  in any case there is typically only 1 component.

Reply:
added a comment describing rational in more detail. i think it he corner case where you had 2 pin enrichments you want to follow the isotope with larger potential temp change, but we could talk more

Reply:
thought about it more and changed to 238. more straightforward to  explain

Reply:
Okay, I think it will work with either isotope, but that sounds good to me.